plugin/src/main/groovy/com/bmuschko/gradle/tomcat/TomcatPlugin.groovy

```diff
@@ -109,6 +109,8 @@ class TomcatPlugin implements Plugin<Project> {
             conventionMapping.map('javaEncoding') { tomcatPluginExtension.jasper.javaEncoding ?: 'UTF8' }
             conventionMapping.map('trimSpaces') { tomcatPluginExtension.jasper.trimSpaces ?: TrimSpaces.TRUE }
             conventionMapping.map('xpoweredBy') { tomcatPluginExtension.jasper.xpoweredBy ?: false }
+            conventionMapping.map('jspFiles') { tomcatPluginExtension.jasper.jspFiles ?: null }
+            conventionMapping.map('failOnError') { tomcatPluginExtension.jasper.failOnError == null ? true : tomcatPluginExtension.jasper.failOnError }
```

Comment:
`tomcatPluginExtension.jasper.failOnError ?: true` is probably the easier expression here.

Reply:
Using the elvis operator (`?:`) will always set the value to true. Since `null` and `false` will cause the operator to return true instead of the provided value.